src/modules/mim/packagemanager.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "PackageManagerModel",
```

Comment:
>PackageManagerModel [](http://example.com/codeflow?start=11&length=19)

"PackageManagerConfigurationModel"

Reply:
Thank you for the feedback, I applied the requested change.